src/SPC/builder/linux/LinuxBuilder.php

```diff
@@ -168,8 +168,11 @@ public function buildPHP(int $build_target = BUILD_TARGET_NONE): void
 
         SourcePatcher::patchBeforeConfigure($this);
 
-        $json_74 = $this->getPHPVersionID() < 80000 ? '--enable-json ' : '';
-        $zts = $this->getOption('enable-zts', false) ? '--enable-zts ' : '';
+        $phpVersionID = $this->getPHPVersionID();
+
+        $json_74 = $phpVersionID < 80000 ? '--enable-json ' : '';
+        $maxExecutionTimers = $this->getOption('enable-zts', false) && $this->getPHPVersionID() > 81000 ? '--enable-zend-max-execution-timers ' : '';
```

Comment:
PHP Version ID seems wrong. (8.1 => `80100`)

And reusing `$phpVersionID` for `$maxExecutionTimers` line?

Reply:
Yes, sorry about that. I'll correct.

Reply:
fixed